packages/utils/src/cmds/schema/diff/index.js

```diff
@@ -0,0 +1,314 @@
+const path = require('path')
+const jsondiffpatch = require('jsondiffpatch')
+const fs = require('fs')
+const utils = require('./utils')
+const ejs = require('ejs')
+const request = require('request')
+const { reporter } = require('@dhis2/cli-helpers-engine')
+const inquirer = require('inquirer')
+
+const defaultOpts = {
+    schemasEndpoint: '/api/schemas.json',
+    infoEndpoint: '/api/system/info.json',
+}
+
+const defaultRequestOpts = {
+    baseUrl: 'https://play.dhis2.org',
+    headers: {
+        'x-requested-with': 'XMLHttpRequest',
+        Authorization: utils.basicAuthHeader('admin', 'district'),
+    },
+    json: true,
+}
+
+let cache
+const prompt = inquirer.createPromptModule({ output: process.stderr })
```

Comment:
A prompt module seems like it belongs in `cli-helpers-engine` so we can be consistent in how we ask for user input.

Reply:
That's for a different PR anyway.